src/main/java/com/shopify/ShopifySdk.java

```diff
@@ -92,6 +88,14 @@
 
 public class ShopifySdk {
```

Comment:
Please fix this code smell:
https://sonarcloud.io/project/issues?branch=feature%2FUpdate-Order&id=com.channelape%3Ashopify-sdk&open=AWrq9TtYvwsQMbwCGWGH&resolved=false&types=CODE_SMELL